cdc/kv/matcher.go

```diff
@@ -26,7 +26,12 @@ func newMatcher() *matcher {
 }
 
 func (m *matcher) putPrewriteRow(row *cdcpb.Event_Row) {
-	m.unmatchedValue[newMatchKey(row)] = row.GetValue()
+	key := newMatchKey(row)
+	value := row.GetValue()
+	if _, exist := m.unmatchedValue[key]; exist && len(value) == 0 {
```

Comment:
Please add some comments to explain why we don't overwrite the old value directly.